asmcli/lib/config.sh

```diff
@@ -26,6 +26,7 @@ configure_package() {
     kpt cfg set asm gcloud.compute.network "${GCE_NETWORK_NAME}"
   else
     kpt cfg set asm gcloud.core.project "${FLEET_ID}"
+    kpt cfg set asm anthos.servicemesh.controlplane.monitoring.enabled "false"
```

Comment:
This part may not be true. In the case ! is_gcp and "${CA}" = "meshca", kpt cfg set asm anthos.servicemesh.controlplane.monitoring.enabled "true". With Hub registration, for onprem, stackdriver is supported.

Reply:
Thanks, I didn't know that! Changed to `if ! is_gcp && ! is_cluster_registered`